tensorboard/plugins/debugger_v2/debugger_v2_plugin_test.py

```diff
@@ -89,6 +96,14 @@ def setUp(self):
         self.plugin = debugger_v2_plugin.DebuggerV2Plugin(context)
         wsgi_app = application.TensorBoardWSGI([self.plugin])
         self.server = werkzeug_test.Client(wsgi_app, wrappers.BaseResponse)
+        # The multiplexer reads data asynchrnously on a separate thread, so
```

Comment:
It’s fine with me to enforce deterministic execution for testing, but
this also has the effect that we’re not testing the code paths the occur
when the reader hasn’t finished updating by the time that the other
methods are invoked, which will certainly occur in real code. Do you
think that it makes sense to add some tests where `run_in_background` is
stubbed out as a no-op rather than a synchronous execution?

(also, sp.: s/`asynchrnously`/`asynchronously`/)

Reply:
I agree with the point that the logic in `run_in_background` ought to be tested. To address this comment, I added a new test file `debug_data_multiplexer_test.py`. The file currently has only a simple assertion, aided by some mocks. This is because the real `run_in_background` method currently just calls `target` once. When it gets more complex, I'll add more mock-based tests in the new file. I added a TODO item to remind myself of that.

Reply:
Sounds reasonable; thanks!